src/app/reservation/create-reservation/reservation-type-step/reservation-type-step.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="h-fit md:h-screen space-y-20">
+  <div class="w-full flex flex-col md:flex-row justify-center items-center space-y-4 md:space-y-0 md:space-x-4 pt-0 md:pt-24">
+    <button class="w-64 h-fit border-2 border-primary-dark-purple rounded-2xl space-y-2 p-2"
+            (click)="setReservationType(ReservationType.FLEXPLEK)"
+            [class.bg-primary-dark-purple]="selectedType === ReservationType.FLEXPLEK">
+      <div class="flex justify-center">
+        <svg xmlns="http://www.w3.org/2000/svg" fill="#5236AA" viewBox="0 0 24 24" height="100" width="100" stroke-width="1.5" stroke="#5236AA"
+             class="size-24"
+             [attr.fill]="selectedType === ReservationType.FLEXPLEK ? '#ffffff' : '#5236AA'"
+             [attr.stroke]="selectedType === ReservationType.FLEXPLEK ? '#ffffff' : '#5236AA'">
+          <path stroke-linecap="round" stroke-linejoin="round" d="M15.75 6a3.75 3.75 0 1 1-7.5 0 3.75 3.75 0 0 1 7.5 0ZM4.501 20.118a7.5 7.5 0 0 1 14.998 0A17.933 17.933 0 0 1 12 21.75c-2.676 0-5.216-.584-7.499-1.632Z" />
+        </svg>
+      </div>
+      <div>
+        <h1 class="font-extrabold text-4xl"
+            [class.text-white]="selectedType === ReservationType.FLEXPLEK">
+          Flexplek
+        </h1>
+      </div>
+    </button>
+    <button class="w-64 h-fit border-2 border-primary-dark-purple rounded-2xl space-y-2 p-2"
+        [class.bg-primary-dark-purple]="selectedType === ReservationType.LOKAAL"
+        (click)="setReservationType(ReservationType.LOKAAL)">
+  <div class="flex justify-center">
+    <svg xmlns="http://www.w3.org/2000/svg" fill="#5236AA" viewBox="0 0 24 24" stroke-width="1.5" stroke="#5236AA"
+         class="size-24"
+         [attr.fill]="selectedType === ReservationType.LOKAAL ? '#ffffff' : '#5236AA'"
+         [attr.stroke]="selectedType === ReservationType.LOKAAL ? '#ffffff' : '#5236AA'">
+      <path stroke-linecap="round" stroke-linejoin="round" d="M18 18.72a9.094 9.094 0 0 0 3.741-.479 3 3 0 0 0-4.682-2.72m.94 3.198.001.031c0 .225-.012.447-.037.666A11.944 11.944 0 0 1 12 21c-2.17 0-4.207-.576-5.963-1.584A6.062 6.062 0 0 1 6 18.719m12 0a5.971 5.971 0 0 0-.941-3.197m0 0A5.995 5.995 0 0 0 12 12.75a5.995 5.995 0 0 0-5.058 2.772m0 0a3 3 0 0 0-4.681 2.72 8.986 8.986 0 0 0 3.74.477m.94-3.197a5.971 5.971 0 0 0-.94 3.197M15 6.75a3 3 0 1 1-6 0 3 3 0 0 1 6 0Zm6 3a2.25 2.25 0 1 1-4.5 0 2.25 2.25 0 0 1 4.5 0Zm-13.5 0a2.25 2.25 0 1 1-4.5 0 2.25 2.25 0 0 1 4.5 0Z" />
+    </svg>
+  </div>
+  <div>
+    <h1 class="font-extrabold text-4xl"
+        [class.text-white]="selectedType === ReservationType.LOKAAL">
+      Lokaal
```

Comment:
Kan je hier 'Vergaderruimte' van maken?

Reply:
Verder moet er in het geval van een vergaderruimte ook de specifieke ruimte gekozen worden, en de ruiloptie zie ik ook nog niet staan.

Reply:
de vergaderruimte reserveren komt later